lib/killbill_client/models/invoice.rb

```diff
@@ -19,17 +19,19 @@ class Invoice < InvoiceAttributes
 
 
       class << self
-        def find_by_id(invoice_id, audit = "NONE", options = {})
+        def find_by_id(invoice_id, with_children_items = false, audit = "NONE", options = {})
           get "#{KILLBILL_API_INVOICES_PREFIX}/#{invoice_id}",
               {
+                  :withChildrenItems => with_children_items,
                   :audit     => audit
               },
               options
         end
 
-        def find_by_number(number, audit = "NONE", options = {})
+        def find_by_number(number, withChildrenItems = false, audit = "NONE", options = {})
```

Comment:
@tungleduyxyz Let's be consistent with name (snake case).

I'm surprised Rubocop didn't catch it 🤔